txstratum/utils.py

```diff
@@ -404,3 +404,23 @@ def _exception_handler(loop: "AbstractEventLoop", context: Dict[str, Any]) -> No
         logger.exception(message, **extra, exc_info=exc_info)
 
     loop.set_exception_handler(_exception_handler)
+
+
+def is_version_gte(version: str, base_version: str) -> bool:
+    """Compare two versions in the format X.Y.Z.
+
+    Validates if version is greater than or equal to base_version
+    """
+    version_split = version.split(".")
+    base_version_split = base_version.split(".")
+
+    assert len(version_split) == len(base_version_split)
+
+    for i in range(len(version_split)):
```

Comment:
I'm afraid you are comparing strings instead of numbers. Maybe just create a `Tuple[int]` and `return version_split >= base_version_split`.

Reply:
You are right. The tuple comparison is interesting, I had forgotten about that, thanks. 

Done here https://github.com/HathorNetwork/tx-mining-service/pull/81/commits/58f622b0d3ac716ca06f0ad4ffe0d23dac6a5d89